exampleConfluence.txt

```diff
@@ -0,0 +1,11 @@
+# This file is an example of what should be found on an adaptor's Confluence page
+# DO NOT CHECK SENSITIVE CONFIG VALUES INTO RESPOSITORY
+
+TEST_ADAPTOR_PORT=3016
+TEST_LEISURE_LINK_API_BASE=
+* Development: https://api-dev.leisurelink.com/v1
```

Comment:
should this have a trailing slash, like the prod URL? (aka https://api-dev.leisurelink.com/v1/)